doc/source/dev/howto_build_docs.rst

```diff
@@ -55,6 +55,18 @@ Doxygen_ can be installed with::
 
     pip install -r requirements/doc_requirements.txt
 
+.. note::
+
+    Since NumPy 2.0 is still in development, so we need to install
```

Comment:
This would also apply to all development versions?

Reply:
This will be necessary until there are releases from the doc dependencies (e.g. scipy, matplotlib, etc.) which support numpy 2.0.

I'd recommend slightly more general wording here, maybe something like:

> It may be necessary to install development versions of the doc
> dependencies to build the docs locally::